src/Modals/ModalPhotoDisplay/ModalPhotoDisplay.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import "./ModalPhotoDisplay.css";
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faAngleLeft, faAngleRight} from '@fortawesome/free-solid-svg-icons'
+import { connect } from 'react-redux';
+import Actions from "../../Actions/actions";
+library.add(faAngleLeft, faAngleRight) 
+
+class ModalPhotoDisplay extends Component {
+    isCurrentClikedImage(image) {
+        return image === this.props.currentClickedImage
+    }
+
+    renderThumbs() {
+        let images = [];
```

Comment:
should be const, it's easy to mix the two up, but remember the difference between re-assignment and mutation.

Re-assignment: you're pointing to a newly allocated object
Mutation: you're pointing to the same object, but the contents of the object change